share/spice/ruby_gems/ruby_gems.js

```diff
@@ -26,13 +26,42 @@
             templates:{
                 group: 'text',
                 detail: false,
-		item_detail: false
+                item_detail: false,
+                variants: {
+                    tile: 'basic1',
+                    tileTitle: '1line',
+                    tileFooter: '2line',
+                    tileSnippet: 'large'
+                },
+                options: {
+                    footer: Spice.ruby_gems.footer
+                }
+            },
+            sort_fields: {
+                downloads: function(a, b) {
+                    return a.downloads > b.downloads ? -1 : 1;
+                }
             },
+            sort_default: 'downloads',
             normalize : function(item){
+                var licenses = [];
+
+                if (item.licenses) {
+                    var keys = Object.keys(item.licenses);
+
+                    for (var i = 0; i < keys.length; i++) {
+                        licenses.push(item.licenses[keys[i]]);
+                    };
```

Comment:
Huh. Looks like they changed the format since the last commit I made here. I'll update the spice.